core/src/epicli/data/common/ansible/playbooks/roles/node_exporter/tasks/main.yml

```
@@ -1,83 +1,113 @@
+        name: node_exporter
+        system: true
+        state: present
 
-- name: Create node_exporter system user
-  user:
-    name: node_exporter
-    system: true
-    shell: "/usr/sbin/nologin"
-    group: node_exporter
-    createhome: false
+    - name: Create node_exporter system user
+      user:
+        name: node_exporter
+        system: true
+        shell: "/usr/sbin/nologin"
+        group: node_exporter
+        createhome: false
 
-- name: Set Node Exporter file name to install
-  set_fact:
-    exporter_file_name: "node_exporter-{{ exporter.version }}.linux-amd64.tar.gz"
+    - name: Set Node Exporter file name to install
+      set_fact:
+        exporter_file_name: "node_exporter-{{ exporter.version }}.linux-amd64.tar.gz"
 
-- name: Download Node Exporter binaries
-  include_role:
-    name: download
-    tasks_from: download_file
-  vars:
-    file_name: "{{ exporter_file_name }}"
+    - name: Download Node Exporter binaries
+      include_role:
+        name: download
+        tasks_from: download_file
+      vars:
+        file_name: "{{ exporter_file_name }}"
 
-- name: Create /opt/node_exporter directories
-  become: true
-  file:
-    path: "{{ item }}"
-    recurse: true
-    owner: root
-    group: "node_exporter"
-    mode: 0750
-    state: directory
-  with_items:
-  - /opt/node_exporter
+    - name: Create /opt/node_exporter directories
+      become: true
+      file:
+        path: "{{ item }}"
+        recurse: true
+        owner: root
+        group: "node_exporter"
+        mode: 0750
+        state: directory
+      with_items:
+      - /opt/node_exporter
 
-- name: Unpack node_exporter binary
-  become: true
-  unarchive:
-    remote_src: true
-    src: "{{ download_directory }}/{{ exporter_file_name }}"
-    dest: "/opt/node_exporter"
-    creates: "/opt/node_exporter/node_exporter"
-    extra_opts: [--strip-components=1]
-    mode: 0755
-    owner: root
-    group: node_exporter
-  check_mode: false
-  notify:
-    - restart prometheus node exporter service
+    - name: Unpack node_exporter binary
+      become: true
+      unarchive:
+        remote_src: true
+        src: "{{ download_directory }}/{{ exporter_file_name }}"
+        dest: "/opt/node_exporter"
+        creates: "/opt/node_exporter/node_exporter"
+        extra_opts: [--strip-components=1]
+        mode: 0755
+        owner: root
+        group: node_exporter
+      check_mode: false
+      notify:
+        - restart prometheus node exporter service
 
-- name: Install node_exporter service to systemd
-  template:
-    src: prometheus-node-exporter.service.j2
-    dest: "/etc/systemd/system/{{ exporter.service.name }}.service"
-    owner: root
-    group: root
-    mode: 0644
+    - name: Install node_exporter service to systemd
+      template:
+        src: prometheus-node-exporter.service.j2
+        dest: "/etc/systemd/system/{{ exporter.service.name }}.service"
+        owner: root
+        group: root
+        mode: 0644
```

How about going human readable? :thinking: 
```suggestion
        mode: u=rw,go=r
```

---

Done